mne/viz/evoked.py

```diff
@@ -277,8 +277,9 @@ def _plot_evoked(evoked, picks, exclude, unit, show, ylim, proj, xlim, hline,
                       plot_type=plot_type)
         _draw_proj_checkbox(None, params)
 
-    for ax in fig.axes[:-1]:
-        ax.set_xlabel('')
+    if len(ch_types_used) > 1:
```

Comment:
if len(ch_types_used) > 1:
is not needed. Just the:

for ax in fig.axes[:len(ch_types_used)-1]: